Meshtastic/Views/Nodes/Helpers/Map/MapContent/MeshMapContent.swift

```diff
@@ -36,14 +42,31 @@ struct MeshMapContent: MapContent {
 
 	var delay: Double = 0
 	@State private var scale: CGFloat = 0.5
+	@State private var connectedNodeCoordinate = LocationsHandler.DefaultLocation
 
 	@MapContentBuilder
 	var positionAnnotations: some MapContent {
+		/// draw lines connecting currentNode
+		if showLinkStrengths {
+			// todo: powersjcb - add config to toggle this - default to off
+
+			if let connectedNodeNum = BLEManager.shared.connectedPeripheral?.num,
```

Comment:
this is apparently not working. Need to figure out the correct way to load the `num` for the the device we are currently connected to.